### ISSUE TITLE
Curious of your usage - care to share?

### ISSUE THREAD

**Issue by jcallaghan** [daweizhangau/esphome_uni_trend_sound_meter#4]
Hi Dawei. Non-issue here. I was curious after finding your project about your configuration/setup?

I am fascinated by weather/environmental related data. During COVID I really got into studying it from the usual weather station data to air quality. I brought a [decibel meter](https://github.com/jcallaghan/home-assistant-config/issues/40) a few years back which I was running during the lockdown in the UK during COVID. I was amazed at how low the environmental noise was due to no traffic on the roads or aircraft in the sky. I recorded all the data to Grafana and InfluxDB and started finding patterns. Sadly, this unit required a USB serial connection which I interfaced with a Raspberry Pi and some Python, but it was difficult to keep this alive when I moved. When I found this project and your component for ESPHome I immediately brought a Ut353bt unit as it piqued my interest again. It fits perfectly in one of these [radiation shields](https://www.amazon.co.uk/gp/product/B017ILZF6C). Interestingly, I setup an ```input_number``` helper in HA which I use to trigger when the unit detects sound above this threshold. This makes for a very interesting security noise sensor!

I was curious if you have this running 24/7? With some good quality batteries, I think I've got max ~72 hours out of it. I plan to wire it to a powered supply but am aware the Bluetooth button sometimes needs to be pressed when powered on for the ESP32 to connect to the unit again. What is your setup and how are you using it?

@jcallaghan

**Comment by daweizhangau** [comment]
@jcallaghan
Hi James,
Thank you for sharing your project and thoughts about this sound meter. 

I bought it for collecting evidence of my neighbor's noise, so that I could make a complaint to my local government with an indication: a list of timestamp and decibel level. (The government officer didn't trust the readings from my Apple Watch.) However my neighbor went quiet before the device arrived. So I was thinking how I could make use of it for something else. If the data can be collected into Home Assistant (HA), it could be more useful, as it support Bluetooth, and doesn't require pairing. That's how I started hacking the protocol and creating this component. 

There are many other options to collect sound level (for example, ESP32 + [Sound Sensor Module](https://www.jaycar.com.au/arduino-compatible-microphone-sound-sensor-module/p/XC4438) + optionally a TTL logic level converter), which could be better than this complicated solution for most home automation scenario. But as [UT353BT](https://meterstest.uni-trend.com/product/ut353-ut353bt/) is a well-calibrated device. If you use it correctly, the result would be more trustworthy. (Not sure putting it in a [radiation shields](https://www.amazon.co.uk/gp/product/B017ILZF6C) would affect the measurement.)

I have not had this device running 24/7. But for I think you just need a UPS or a power supply with battery. Just be aware that many commercial power banks turn off automatically for small current. Or you just set up a notification for data stream interruption, and go to press that BT button.

BTW, this brand has [wind speed sensor](https://meterstest.uni-trend.com/product/ut363-ut363bt/) as well. I do not have it, but I guess it would be similar to pull its data.

**Comment by adizanni** [comment]
Hello @daweizhangau, I use it since I installed air conditioner to be sure that the compressor noise is not affecting my neighbors. I connected it to a power supply with a dc transformer, and it is working flowlessly since 3 months without errors. Thanks a lot !

**Comment by daweizhangau** [comment]
@adizanni I am really glad to hear your story!

**Comment by CurlyWirly** [comment]
Hi Dawei and thanks for this project. I needed a way to capture plane noise levels as they flew near our house. Our local airport recently changed their flight paths without any consultation. This has caused a lot of suffering for the residents that are now impacted by large jet aircraft departing and approaching the airport as they fly over their once quite neighbourhoods. I was looking for a sound level meter that could work with Home Assistant, be cost effective and relatively accurate. I found your project and as I had set up other esphome devices before, I bought the UT353BT and got it all setup.
I have it running 24/7 by wiring up a 5v USB adapter and it continuously sends Db readings to my Home Assistant. I also use the FlightRadar24 integration which logs flights within an area around my house.
The final piece I am now setting up is an automation that is triggered by a plane entering my area as registered within FlightRadar24 and a noise level above a threshold as recorded with your project. This will then send the data of that moment to a google sheet using the Google Sheets integration. Each new row will be a flight that has been close to my house and above a reasonable noise level.
I'm almost done now, just stuck on the correct yaml for flightradar24 needed to grab the data specific to each cell.
